lib/api.js

```diff
@@ -152,22 +152,15 @@ function shellScriptWorker(script = "worker.sh", options = {}) {
 
 async function buildTransformers(renditionCallback, options={}) {
     const transformers = [];
-
-    if (!options.manifests) {
+    if (!options.manifest) {
         return transformers;
     }
-
-    const manifests = options.manifests;
-    console.log('Manifest array', manifests);
-
-    manifests.forEach(oneManifest => {
-        const oneTransformer = new WorkerCallbackTransformer(renditionCallback, oneManifest);
-        debug(`Created a transformer named ${oneTransformer.name} with manifest:`);
-        debug(oneManifest);
-        transformers.push(oneTransformer);
-    });
-    return transformers.concat(options.transformerCatalog);
     
+    debug('Manifest', options.manifest);
+    const transformer = new WorkerCallbackTransformer(renditionCallback, options.manifest);
+    debug(`Created a transformer named ${transformer.name}`);
+    transformers.push(transformer);
+    return transformers.concat(options.transformerCatalog);
```

Comment:
Alternative:
```suggestion
    return transformers.concat(transformer, options.transformerCatalog);
```
(concat can take many params)

Reply:
got that at line 161.